app/controllers/posts_controller.rb

```diff
@@ -1,36 +1,28 @@
 class PostsController < ApplicationController
   layout 'standard'
-
   def index
     @user = User.find(params[:user_id])
     @posts = Post.where(author_id: params[:user_id]).order(id: :asc)
```

Comment:
- [ ] In order to effectively resolved the N+1 query issue, let's kindly make sure we fetch each post for a user together with their comments. The best way to approach the issue is to do something like this:

`@posts = @user.posts.includes(:comments)`

Reply:
thank you for your review @FrederickMih , I have made the required changes.